### ISSUE TITLE
Seeing an exception on develop when running `salt '*' test.ping` or other execution modules

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt#35582]
### Description of Issue/Question
On the head of develop I am seeing the following:

```
[ch3ll@thecakeisalie salt-pkg-tests]$ sudo salt '*02' test.version
ch3ll-minion02:
    2016.3.0-1184-g3863045
Exception AttributeError: "'NoneType' object has no attribute 'close'" in <bound method IPCMessageSubscriber.__del__ of <salt.transport.ipc.IPCMessageSubscriber object at 0x7f8ef38b1ad0>> ignored
```

### Setup
Master and Minion both on develop

### Steps to Reproduce Issue
1. `salt '*' test.version`

### Versions Report
`2016.3.0-1184-g3863045`

**Comment by Ch3LL** [comment]
versions report:

```
[ch3ll@thecakeisalie salt-pkg-tests]$ sudo salt --versions-report
Salt Version:
           Salt: 2016.3.0-1184-g3863045
 
Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.4.2
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.3
        libgit2: 0.22.0
        libnacl: Not Installed
       M2Crypto: 0.21.1
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.10
       pycrypto: 2.6.1
         pygit2: 0.22.0
         Python: 2.7.10 (default, Jun 20 2016, 14:45:40)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.7.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: fedora 22 Twenty Two
        machine: x86_64
        release: 4.4.14-200.fc22.x86_64
         system: Linux
        version: Fedora 22 Twenty Two

```

**Comment by Ch3LL** [comment]
fixed by #35478
